Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -46,7 +46,7 @@
         locals()["_electronSeedProducers"+_postfix] = ["tripletElectronSeeds"]
 
 _removeForFastSimSeedProducers =["initialStepSeedsPreSplitting",
-                                 "jetCoreRegionalStepSeeds",
+                                 "jetCoreRegionalStepSeedsBarrel","jetCoreRegionalStepSeedsEndcap",
```

Comment:
shouldn't this be an extension instead of the replacement (at least until `seedingDeepCore` becomes the default)?

Reply:
https://github.com/cms-sw/cmssw/pull/33531/files#r625269801

please comment

Reply:
this one is apparently resolved (with `jetCoreRegionalStepSeeds` now kept in place) ; thanks.